package.json

```diff
@@ -33,25 +33,20 @@
   "homepage": "https://github.com/silverstripe/silverstripe-campaign-admin",
   "dependencies": {
     "@popperjs/core": "^2.11.6",
-    "bootstrap": "^4.6.2",
+    "bootstrap": "^5.2.0",
     "classnames": "^2.3.2",
     "core-js": "^3.26.0",
     "deep-freeze-strict": "^1.1.1",
     "prop-types": "^15.8.1",
-    "react": "^18.2.0",
```

Comment:
Removed all of the `react*` deps because they're not needed, and also because I could ran `yarn build` with them here as they causes the 'Invalid hook call' issue.

Reply:
I assume you mean "because I _couldn't_ run `yarn build` with them there"